dev/jenkins/run_docker.sh

```diff
@@ -55,6 +55,7 @@ function main {
   run_args+=" -e MAVEN_CONFIG=${home}/.m2"
 
   run_args+=" -e ALLUXIO_USE_FIXED_TEST_PORTS=true"
+  run_args+=" -e ALLUXIO_PORT_COORDINATION_DIR=${home}"
```

Comment:
is there a downside to doing `${home}` vs `${home}/.alluxio_port_coordination` or something of the sort?